app/views/dashboard/ng-partials/sequence-builder/_move_abs.html.haml

```diff
@@ -0,0 +1,91 @@
+.small-12.columns
+  .small-12.columns.blue-block.block
+    .row.blue-header{"as-sortable-item-handle" => "", "data-topbar" => ""}
+      .small-12.columns
+        %nav.row.top-bar.blue-header{:role => "navigation"}
+          %ul.title-area
+            %li.name
+              %h5 MOVE ABSOLUTE
+          .icon-group
+            %a{:href => ""}>
+              %i.fa.fa-angle-up
+              %a{:href => ""}
+                %i.fa.fa-trash{"ng-click" => "deleteStep($index)"}
+              %a{:href => ""}
+                %i.fa.fa-clipboard{"ng-click" => "copy(step, $index)"}
+            %a{:href => ""}
+              %i.fa.fa-bars{"as-sortable-item-handle" => ""}
+    .row.content-area.blue-content
+      .large-8.columns
+        .row.collapse.prefix-radius
+          .small-2.columns
+            %span.prefix.uppercase x
+          .small-8.columns
+            %input{:placeholder => "Inherit Plant-ID_X", :type => "text"}
+          .small-2.columns
+            %button.tiny.button.postfix.dropdown.light-gray{"aria-controls" => "drop1", "aria-expanded" => "false", "data-dropdown" => "drop1", :href => "#"}
+            %ul#drop1.f-dropdown{"aria-hidden" => "true", "data-dropdown-content" => "", :tabindex => "-1"}
```

Comment:
We will most likely need to use something different than this (because we're using angular instead of plain JS/ Jquery), but it will be good enough for the sake of creating an HTML mockup.